Aseprite.js

```diff
@@ -288,21 +322,40 @@ class Aseprite {
     this.skipBytes(7);
     const w = this.readNextWord();
     const h = this.readNextWord();
+    const chunkInfo = { layerIndex, xpos: x, ypos: y, opacity, celType, w, h };
+    if (celType === 0 || celType === 2) {
+      return this.readImageCelChunk(chunkSize, chunkInfo)
+    }
+    if (celType === 3) {
+      return this.readTilemapCelChunk(chunkSize, chunkInfo)
+    }
+  }
+  readImageCelChunk(chunkSize, chunkInfo) {
     const buff = this.readNextRawBytes(chunkSize - 26); //take the first 20 bytes off for the data above and chunk info
     let rawCel;
     if(celType === 2) {
       rawCel = zlib.inflateSync(buff);
     } else if(celType === 0) {
       rawCel = buff;
     }
-    return { layerIndex,
-      xpos: x,
-      ypos: y,
-      opacity,
-      celType,
-      w,
-      h,
-      rawCelData: rawCel}
+    return { ...chunkInfo, rawCelData: rawCel };
```

Comment:
Why would you pass `chunkInfo` just to return it in the same function? (using the spread operator, but still returning it).

Couldn't you just return it in `readCelChunk`?

Reply:
Fixed in b11bcb1.